modules/apps/transfer/types/keys.go

```diff
@@ -38,6 +34,18 @@ const (
 	ParamsKey = "params"
 )
 
+const (
+	// Version defines the current version the IBC transfer
+	// module supports
+	CurrentVersion = "ics20-2"
```

Comment:
This should just be `Version` in my opinion like the godoc suggests. External teams are likely using `Version` as a constant and this will break their code?